README.md

```diff
@@ -16,6 +16,24 @@ The result of the sync is logged to a sync log available from the settings scree
 
 Only simple products are synced. Configurable products are not synced.
 
+## Requirements
+
+In order for the cron to successfully process the listings sync in parallel-thread manner, the magento cron needs to be
+declared in the crontab via the following:
+
+* * * * * php /path/to/magento/cron.php -mdefault 1
+
+or
+
+* * * * *  /bin/sh /path/to/magento/cron.sh cron.php -mdefault 1 > /dev/null 2>&1 &
+
```

Comment:
Is there any preferred way to do this? Is the magento one safer or more brittle across versions? Why would I use one vs the other?